dbt-tests-adapter/dbt/tests/adapter/incremental/fixtures.py

```diff
@@ -303,3 +303,34 @@
 
 from source_data
 """
+
+_MODELS__SRC_ARTISTS = """
+{{
+    config(
+        materialized='table',
+    )
+}}
+
+{% if var("version", 0) == 0 %}
+
+    select {{ dbt.current_timestamp() }} as inserted_at, 'john' as name
+
+{% else %}
+
+    -- add a non-zero version to the end of the command to get a different version:
+    -- --vars "{'version': 1}"
+    select {{ dbt.current_timestamp() }} as inserted_at, 'john' as Name, 'engineer' as "Job"
```

Comment:
I wonder if using [`adapter.quote`](https://docs.getdbt.com/reference/dbt-jinja-functions/adapter#quote) and/or [`string_literal`](https://docs.getdbt.com/reference/dbt-jinja-functions/cross-database-macros#string_literal) would enable this test to work across more adapters without modification?
 
```suggestion
    select {{ dbt.current_timestamp() }} as inserted_at, {{ dbt.string_literal("john") }} as Name, {{ dbt.string_literal("engineer") }} as {{ adapter.quote("Job") }}
```